config/toml.go

```diff
@@ -345,6 +345,7 @@ dial_timeout = "{{ .P2P.DialTimeout }}"
 # Mempool version to use:
 #   1) "v0" - (default) FIFO mempool.
 #   2) "v1" - prioritized mempool.
+#   3) "v2" - content addressable transaction pool
```

Comment:
similar 
```suggestion
#   3) "v2" - (default) content addressable transaction pool
```

and  `1) "v0" - (default) FIFO mempool.` =>  `1) "v0" - FIFO mempool.`